app/screens/end-normal-game.js

```diff
@@ -16,159 +31,196 @@ class EndNormalGame extends React.PureComponent {
     };
     this.setModalSignIn = this.setModalSignIn.bind(this);
     this.setModalSignUp = this.setModalSignUp.bind(this);
-    this.setModalHide = this.setModalHide.bind(this);
+    this.setModalHide = this.setModalHide.bind(this);  //si uso el this tengo que bindear
+    this.renderLeaderBoard = this.renderLeaderBoard.bind(this);
+    this.saveMatch = this.saveMatch.bind(this);
   }
 
   setModalSignIn() {
-    this.setState({ showModal: 'signIn' });    
+    this.setState({ showModal: 'signIn' });
   }
 
   setModalSignUp() {
     this.setState({ showModal: 'signUp' });
   }
 
   setModalHide() {
-    this.setState({ showModal: 'hide' });    
+    this.setState({ showModal: 'hide' });  
+  }
+
+  saveMatch() {
+    matchService.update(this.props.matchData.match)
+      .then((res) => {
+        this.props.history.push('/');
+      })
+      .catch((err) => {
+        alert(err.error);
+      });
+  }
+
+  renderLeaderBoard() {
+    var init_leaderboard = this.props.matchData.match.game.ranking.slice(); //clean copy of ranking
+    init_leaderboard.push({ user: this.props.matchData.state.player,
+                       points: this.props.matchData.state.score
+                    });
+
+    var leaderboard = sortBy(init_leaderboard, 'points').reverse();
+
+
+    var lastIndex = size(leaderboard) - 1;
+    var items = [];
+
+
+    var i = 0;
+    var encontre = false;
+    while (!encontre && i <= lastIndex) {
+      if (this.props.matchData.state.player === leaderboard[i].user && 
+        this.props.matchData.state.score === leaderboard[i].points) {
+          encontre = true;
+      }
+      else {
+        i = i + 1;
+      }
+    }
+
+    var userPlace = -1;
+    if (encontre) {
+      userPlace = i;
+    } 
+
+    if (userPlace > -1) {  //check if player was found
+      
+      if (lastIndex <= 4) {
+
+        //show until 5 users starting from 0  
+        for (i = 0; i <= lastIndex; i++) {
+          items.push(
+            isEqual(leaderboard[i].user, this.props.matchData.state.player) ? ( 
+            <tr className='current-player'>
+              <td>{ i + 1 }</td>
+              <td>{ leaderboard[i].user }</td>
+              <td>{ leaderboard[i].points } pts</td>
+            </tr>
+            ) : (
+              <tr>
+                <td>{ i + 1 }</td>
+                <td>{ leaderboard[i].user }</td>
+                <td>{ leaderboard[i].points } pts</td>
+              </tr>
+            )
+          )
+        }
+
+      }
+      else {
+        if (userPlace === 0 || userPlace === 1) { //if current user is in first place or second one
```

Comment:
poner los comentarios en la linea anterior. De la guia de estilos:
// bad
const active = true;  // is current tab

// good
// is current tab
const active = true;